src/components/Header/index.tsx

```diff
@@ -63,6 +63,15 @@ const Header = ({
                 Download
               </a>
             </li>
+            <li className="nav__tabs">
+              <a
+                href="/community"
```

Comment:
We should use `<Link>`here
The reason docs and download use href is because the new versions are not published yet.